store/conf/attrs/zimbra-attrs.xml

```diff
@@ -5273,7 +5273,7 @@ TODO: delete them permanently from here
   <desc>sieve script generated from user outgoing filter rules</desc>
 </attr>
 
-<attr id="1131" name="zimbraOAuthConsumerCredentials" type="ostring" cardinality="multi" optionalIn="globalConfig,domain" flags="domainInherited" since="7.0.0">
+<attr id="1131" name="zimbraOAuthConsumerCredentials" type="ostring" cardinality="multi" optionalIn="globalConfig,domain" flags="domainInherited,domainInfo" since="7.0.0">
```

Comment:
isn't this a security issue?

Reply:
I have added this for testing for now 
will remove later